scripts/utils-wolfssl.sh

```diff
@@ -59,8 +60,8 @@ install_wolfssl() {
             WOLFSSL_CONFIG_CPPFLAGS=CPPFLAGS="-I${OPENSSL_INSTALL_DIR} -DHAVE_AES_ECB -DWOLFSSL_AES_DIRECT -DWC_RSA_NO_PADDING -DWOLFSSL_PUBLIC_MP -DECC_MIN_KEY_SZ=192 -DHAVE_PUBLIC_FFDHE -DHAVE_FFDHE_6144 -DHAVE_FFDHE_8192 -DFP_MAX_BITS=16384 -DWOLFSSL_DH_EXTRA -DWOLFSSL_PSS_LONG_SALT -DWOLFSSL_PSS_SALT_LEN_DISCOVER"
         fi
 
-        ./autogen.sh &>> $LOG_FILE
-        ./configure ${WOLFSSL_CONFIG_OPTS} "${WOLFSSL_CONFIG_CPPFLAGS}" -prefix=${WOLFSSL_INSTALL_DIR} &>> $LOG_FILE
+        ./autogen.sh 2>&1 >> $LOG_FILE
+        ./configure ${WOLFSSL_CONFIG_OPTS} "${WOLFSSL_CONFIG_CPPFLAGS}" -prefix=${WOLFSSL_INSTALL_DIR} 2>&1 >> $LOG_FILE
```

Comment:
```
In scripts/utils-wolfssl.sh line 63:
        ./autogen.sh 2>&1 >> $LOG_FILE
                     ^--^ SC2069 (warning): To redirect stdout+stderr, 2>&1 must be last (or use '{ cmd > file; } 2>&1' to clarify).
                             ^-------^ SC2086 (info): Double quote to prevent globbing and word splitting.
```
```
In scripts/utils-wolfssl.sh line 64:
        ./configure ${WOLFSSL_CONFIG_OPTS} "${WOLFSSL_CONFIG_CPPFLAGS}" -prefix=${WOLFSSL_INSTALL_DIR} 2>&1 >> $LOG_FILE
                    ^--------------------^ SC2086 (info): Double quote to prevent globbing and word splitting.
                                                                                ^--------------------^ SC2086 (info): Double quote to prevent globbing and word splitting.
                                                                                                       ^--^ SC2069 (warning): To redirect stdout+stderr, 2>&1 must be last (or use '{ cmd > file; } 2>&1' to clarify).
                                                                                                               ^-------^ SC2086 (info): Double quote to prevent globbing and word splitting.
```